src/dynamicprompts/parser/parse.py

```diff
@@ -101,7 +102,7 @@ def _configure_literal_sequence(
         non_literal_chars += rf"|${parser_config.variant_end}"
 
     non_literal_chars = re.escape(non_literal_chars)
-    literal = pp.Regex(rf"((?!{double_underscore})[^{non_literal_chars}])+")(
+    literal = pp.Regex(rf"((?!{parser_config.wildcard_wrap})[^{non_literal_chars}])+")(
```

Comment:
This will probably need to be `re.escape`d – otherwise a wildcard wrap like `..` would cause havoc here.
```suggestion
    literal = pp.Regex(rf"((?!{re.escape(parser_config.wildcard_wrap)})[^{non_literal_chars}])+")(
```

Reply:
OK This is the problem I overlooked. I'm not very familiar with regular expressions.
I will try to write a test. 😺